content/powershell/concepts/operators/operators.md

```diff
@@ -1,14 +1,164 @@
+|   `/=`   |    Division Compound Assignment    | `$x /= 3` is short for `$x = $x / 3`.       |
+|   `%=`   |     Modulo Compound Assignment     | `$x %= 3` is short for `$x = $x % 3`.       |
+
+### Example
+
+```shell
+PS > $number = 4
+PS > $number += 6 # $number is now 10
+PS > $number /= 2 # $number is now 5
+PS > $number
+5
+```
+
+## Unary Operators
+
+Unary operators increase or decrease the value of a variable by 1.
+
+| Operator |      Name      | Description                                                   |
+| :------: | :------------: | ------------------------------------------------------------- |
+|   `++`   |   Increment    | `$x++` returns `$x + 1`                                       |
+|   `--`   |   Decrement    | `$x--` returns `$x - 1`                                       |
+
+## Equality Comparison Operators
+
+Equality operators in PowerShell are binary operators that compare two integer or string values that return `True` if the operator condition is met, otherwise `False`.
+
+| Operator |           Name           | Description                                                   |
+| :------: | :----------------------: | ------------------------------------------------------------- |
+|   `-eq`  |          Equal           | `$x -eq $y` is `true` if `x` and `y` are equal.               |
+|   `-ne`  |        Not Equal         | `$x -ne $y` is `true` if `x` and `y` are not equal.           |
+|   `-gt`  |        Less Than         | `$x -gt $y` is `true` if `x` is less than `y`.                |
+|   `-lt`  |       Greater Than       | `$x -lt $y` is `true` if `x` is greater than `y`.             |
+|   `-ge`  |  Less Than or Equal to   | `$x -ge $y` is `true` if `x` is less than or equal to `y`.    |
+|   `-le`  | Greater Than or Equal to | `$x -le $y` is `true` if `x` is greater than or equal to `y`. |
+
+## Logical Operators
+
+Logical operators allow us to combine multiple operator expressions and statements into complex conditionals. They operate on boolean values and return boolean values.
+
+| Operator            | Name  | Description                                                           |
+| :-----------------: | :---: | --------------------------------------------------------------------- |
+|   `-and`            | And   | `$x -and $y` is `True` only if `$x` and `$y` are both `True`.         |
+|  `-or`              | Or    | `$x -or $y` is `True` if either `$x` or `$y` is `True`.               |
+|  `-xor`             | Xor   | `$x -xor $y` is `True` if only, but not both, `$x` or `$y` is `True`. |
+|   `!` or `-not`     | Not   | `!$x` is `True` when `$x` is `False` and `False` when `$x` is `True`. |
+
+## Operator Precedence
+
+Precedence order is the order in which PowerShell evaluates the operators if multiple operators are used in the same expression. Operator precedence in PowerShell is as follows:
+
+- Parentheses: `( )`
```

Comment:
I think this would do well as a numbered list

Reply:
Done in abc2e07349b7dd732437fe653ec310aefbd5d79a